templates/frontend/components/loginForm.tpl

```diff
@@ -41,40 +41,31 @@
 			</label>
 			<input type="password" class="form-control" name="password" id="passwordModal" value="{$password|escape}"
 			       maxlength="32" required>
-		</div>
-		<div class="row">
-			<div class="col-md-6">
-				<div class="form-group form-group-forgot">
-					<small class="form-text">
-						<a href="{url page="login" op="lostPassword"}">
-							{translate key="user.login.forgotPassword"}
-						</a>
-					</small>
-				</div>
-			</div>
-			<div class="col-md-6">
-				<div class="custom-control custom-checkbox">
-					<input type="checkbox" class="custom-control-input" name="remember" id="rememberModal" value="1"
-					       checked="$remember">
-					<label for="rememberModal" class="custom-control-label">
-							{translate key="user.login.rememberUsernameAndPassword"}
-					</label>
-				</div>
+
+			<div class="custom-control custom-checkbox">
+				<input type="checkbox" class="custom-control-input" name="remember" id="rememberModal" value="1"
+				       checked="$remember">
```

Comment:
Should this be on a second line?

Reply:
Thanks for catching that - it was already like that before I moved around the markup, maybe because that line was too long (?). I’m changing it on the same line -- but this reminds me: do you think it’s a good idea to impose a max line length (e.g. 80 characters max) for templates in order to keep the HTML readable?

Reply:
If someone can find a linter/formatter for smarty files, I'd happily apply it. We use `prettier` for the ui-library to enforce a consistent style. 

But unless it's automatic and applied against commits (rather than relying on someone running it manually) it is hard to police consistently.